### DISCUSSION TITLE
Layer Manager setOpen method undefined

### DISCUSSION THREAD

**Discussion by singh-jp-7** [GrapesJS/grapesjs#5661] [Q&A]
I have Layer Manager in my project but I'm not able to use methods like isOpen and setOpen for layers. When I log editor['Layers'] to my console, I get methods like getConfig(), onLoad(), render(), postRender(), getRoot(), setRoot(), but the setOpen method is not there. which is there in the documentation So any lead about that, because of that I'm not able to control opening and closing ofm y layers.

**Comment by artf** [comment]
Use the latest version
